ga4gh/sqliteBackend.py

```diff
@@ -0,0 +1,112 @@
+"""
+Library for accessing SQLite-backed data
+optimized for typical REST queries as defined by the GA4GH server.
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import sqlite3
+
+
+def sqliteRows2dicts(sqliteRows):
+    """
+    unpacks sqlite rows as returned by fetchall
+    into an array of simple dicts.
+    :param sqliteRows: array of rows returned from fetchall DB call
+    :return:  array of dicts, keyed by the column names.
+    """
+    return map(lambda r: dict(zip(r.keys(), r)), sqliteRows)
+
+
+def sqliteRow2Dict(sqliteRow):
+    """
+    As above, but for a single row
+    :param sqliteRow: single row returned from fetchone DB call
+    :return: dictionary corresponding to this row
+    """
+    return dict(zip(sqliteRow.keys(), sqliteRow))
+
+
+def limitsSql(pageToken=0, pageSize=None):
+    """
+    Takes parsed pagination data, spits out equivalent SQL 'limit' statement.
+    :param pageToken: starting row position, int or string containing an int
+    :param pageSize: number of records requested for this transaciton,
+    int or string containing an int
+    :return: SQL 'limit' statement string to append to query.
+    """
+    if pageToken is None:
+        pageToken = 0
+    if pageSize is not None or pageToken > 0:
+        start = int(pageToken)
+        end = start + int(pageSize)
+        return " LIMIT {}, {}".format(start, end)
+    else:
+        return ""
+
+
+def _whereClauseSql(**whereClauses):
+    """
+    Takes parsed search query parameters,
```

Comment:
Don't need to indent when breaking docstring descriptions (you do for sphinx params though).